docs/runbooks/rob-1064-h6-seal-verify-reseal.md

```
@@ -0,0 +1,253 @@
+            digest({k: v for k, v in c.items() if k != "fold_id"}))
+        obs[(c["family"], c["config_id"])].add(c["observation_count"])
+    payloads = {d for v in per_config.values() for d in v}
+    print(f"{path.name}: cells={len(cells)} distinct_payloads={len(payloads)} "
+          f"per_config_distinct={sorted({len(v) for v in per_config.values()})} "
+          f"per_config_distinct_observation_count={sorted({len(v) for v in obs.values()})}")
+PY
+```
+
+Expected:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)